src/json/members/jessez.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Jesse Zong",
+    "pennkey": "jessez",
+    "major": "Computer Science",
+    "school": "Engineering",
+    "bio": "Hi, I'm Jesse! I'm a CIS major in engineering",
+    "hometown": "Ellicott City, Maryland",
+    "team": "Penn Mobile",
+    "roles": ["Backend Engineer"],
+    "photo": "https://i.imgur.com/1p5qwK5.jpg",
```

Comment:
Because this image is not a square, it's gonna get cropped. I'd recommend cropping it yourself first.

Reply:
thanks! i've changed the image to reflect that.